data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -219,6 +219,33 @@
     "Michigan EJSCREEN Priority Community"
 )
 
+# Maryland EJSCREEN Data.
+MARYLAND_EJSCREEN_TRACT_25_PERCENT_FIELD: str = (
+    "Tract is >=25% all other Maryland Tracts"
+)
+MARYLAND_EJSCREEN_TRACT_50_PERCENT_FIELD: str = (
+    "Tract is >=50% all other Maryland Tracts"
+)
+MARYLAND_EJSCREEN_TRACT_75_PERCENT_FIELD: str = (
+    "Tract is >=75% all other Maryland Tracts"
+)
+MARYLAND_EJSCREEN_TRACT_90_PERCENT_FIELD: str = (
+    "Tract is >=90% all other Maryland Tracts"
+)
+
+MARYLAND_EJSCREEN_PERCENTILE_FIELD: str = (
```

Comment:
I would suggest replacing this with simply referencing `field_names.MARYLAND_EJSCREEN_SCORE_FIELD + field_names.PERCENTILE_FIELD_SUFFIX` everywhere, to follow the pattern used by other code and have a standardized field name.

Reply:
👍

Reply:
My only concern is that _percentile_ has a very different meaning than _percentile rank_ - as raised in #239. Is this not misleading?

Reply:
We use "percentile" to mean what you're calling "percentile rank" everywhere in this codebase, and it's very standard usage. 

From wikipedia: 

> In statistics, a k-th percentile, is a score below which a given percentage k of scores in its frequency distribution falls or a score at or below which a given percentage falls. For example, the 50th percentile is the score below which or at or below which 50% of the scores in the distribution may be found. 

The percentile is K. 

Let's go ahead with calling these percentiles, as we do everywhere else.

Reply:
@lucasmbrown-usds : This definition is  aligned with what I shared and provided [here](https://github.com/usds/justice40-tool/issues/239#issuecomment-1003567593). This is the main area of concern. I note this in the codebase. Just because we use it everywhere else doesn't make this necessarily a correct statement. 

Quoting my reasoning from that link as well:

_A statement such as "a height of 173 cm has a percentile rank of 80" has a different meaning from a statement such as "the 80th percentile is 173 cm."_

_The first of these two statements say that 80% of the population had heights that were less than a height of 173 cm, while the second statement says that 80 percent of the population had a height of 173 cm or less. Statements involving nth percentiles, which are not employed in this case, are typically used for defining the cutoffs when the scores have been arranged numerically into 100 equal intervals._

_In summary, percentile refers to a single point; a percentile rank covers an entire interval. When we start with a given proportion or percentage, we compute a percentile whose value is some raw score; when we start with a given raw score or interval, we compute a percentile rank whose value is between 0 and 100._

Reply:
The CDC SVI Index also references percentile ranks as well in their methodology. I am just not comfortable with changing this to something that does not reflect what was actually done to the data. 

I can just remove that percentile rank reference altogether and use the reported EJ Score (which is a percentile).

Reply:
> We use "percentile" to mean what you're calling "percentile rank" everywhere in this codebase, and it's very standard usage.
> 
> From wikipedia:
> 
> > In statistics, a k-th percentile, is a score below which a given percentage k of scores in its frequency distribution falls or a score at or below which a given percentage falls. For example, the 50th percentile is the score below which or at or below which 50% of the scores in the distribution may be found.
> 
> The percentile is K.
> 
> Let's go ahead with calling these percentiles, as we do everywhere else.

The percentile is `K` as it pertains to what? A single entity or an interval? My answer in the issue addresses this.

In the same article that you cite, I should have added that there is a section in `Definitions` with the statement `There is no standard definition of percentile`.